_resources/participatory-design-inclusivity-heuristics.md

```diff
@@ -0,0 +1,248 @@
+---
+layout: heuristics
+title: Participatory design&#58; inclusivity heuristics
+permalink: /resources/participatory-design-inclusivity-heuristics/
+redirect_from:
+  - /participatory-design-inclusivity-heuristics/
+description: A set of heuristics for ensuring that a participatory design effort is proceeding in an inclusive way.
```

Comment:
If the title has words that might be unfamiliar to non-experts, I try to write the description so that it helps readers figure out what I mean. In this case, I'd probably substitute `guidelines` for `heuristics`, for example. `Inclusive` and `participatory` might also be good words to define.

Reply:
@esgoodman great suggestions! Will give this another pass.